src/components/common/Footer/index.js

```diff
@@ -4,20 +4,107 @@ import Link from 'gatsby-link';
 import withStyles from 'react-jss';
 import cx from 'classnames';
 
+import navigation from '../../../data/navigation.json';
 import logoReversed from '../../../assets/images/rwr/logo-mini-reversed.png';
+import youtube from '../../../assets/images/footer/youtube.png';
+import facebook from '../../../assets/images/footer/facebook.png';
+import twitter from '../../../assets/images/footer/twitter.png';
+import meetup from '../../../assets/images/footer/meetup.png';
 import styles from './styles';
+import { SignUp, Contact } from '../../../containers';
 
 const Footer = props => {
-  const { theme, sheet, classes, className, ...etc } = props;
+  const { theme, sheet, classes, className, showContact, ...etc } = props;
   return (
     <footer className={cx(classes.root, 'container', className)} {...etc}>
       <div className="row">
-        <div className="col-xs-12">
-          <img className={classes.logo} src={logoReversed} />
+        {showContact && (
```

Comment:
Let's better verify if in the component props comes the Contact component. If not, don't show it.